### ISSUE TITLE
GetAvailability - Empty workweek in calendar settings breaks the query

### ISSUE THREAD

**Issue by LasseHynninen** [OfficeDev/ews-managed-api#51]
When user has changed their workweek period from the calendar settings, any GetAvailability() request seems to fail.

Workweek setting seen from OWA 2013:
![none](https://cloud.githubusercontent.com/assets/11871875/8700272/83735c50-2b13-11e5-8131-9edf6b6cc427.png)

Normal workweek in the 'GetUserAvailabilityResponse':
```xml
<WorkingPeriodArray>
    <WorkingPeriod>
    <DayOfWeek>Monday Tuesday Wednesday Thursday Friday</DayOfWeek>
    <StartTimeInMinutes>480</StartTimeInMinutes>
    <EndTimeInMinutes>1020</EndTimeInMinutes>
    </WorkingPeriod>
</WorkingPeriodArray>
```

And same when the enduser has changed their settings:
```xml
<WorkingPeriodArray>
    <WorkingPeriod>
    <DayOfWeek />
    <StartTimeInMinutes>480</StartTimeInMinutes>
    <EndTimeInMinutes>960</EndTimeInMinutes>
    </WorkingPeriod>
</WorkingPeriodArray>
```

GetAvailability() fails in Core\EwsUtilities.cs, line 664:
```cs
string[] enumValues = value.Split(separators);
```
Exception is a normal NullReference Exception, since the 'value' is indeed null.

Caller is in ComplexProperties\Availability\WorkingPeriod.cs, TryReadElementFromXml():
```cs
            switch (reader.LocalName)
            {
                case XmlElementNames.DayOfWeek:
                    EwsUtilities.ParseEnumValueList<DayOfTheWeek>(
                        this.daysOfWeek,
                        reader.ReadElementValue(),
                        ' ');
                    return true;
```
reader.ReadElementValue(), :point_left: This reads null from the XML when the user has unchecked their days in the workweek.

**Comment by dclaux** [comment]
Thanks for fixing this.

**Comment by LeightonSean** [comment]
I'm not experiencing this fix in the latest NuGet package (2.2.0) but I could resolve my issue by getting the latest source code and building the DLL from there.

**Comment by HenningOI** [comment]
Why is there no new nuget version that fixes this error?
